src/metrics/review-latency.ts

```diff
@@ -148,7 +148,7 @@ type Review = {
  * must not be from the pull request author and must be from different
  * reviewers.
  */
-function getReviewsForPullRequest(pullRequest: PullRequest): Review[] {
+export function getReviewsForPullRequest(pullRequest: PullRequest): Review[] {
```

Comment:
Factor out to a shared file? Maybe `common-queries` should just be `common` for now.

Reply:
Done, done.